blocks/results-list-block/features/results-list/results/helpers.js

```diff
@@ -0,0 +1,49 @@
+const defaultResultList = {
+  content_elements: [],
+};
+
+const reduceResultList = (state = defaultResultList, action) => {
+  const { type, data } = action;
+
+  if (!data) { return state; }
+
+  const oldItemIds = state.content_elements.map((element) => element._id);
+  const uniqueNewItems = data.content_elements
+    .filter((element) => !oldItemIds.includes(element._id));
```

Comment:
is there fallback if there's no content_elements or an _id. I don't remember this functionality from before

Reply:
You mean a fallback for elements without an _id?  From what I can tell, that's not possible?

Reply:
As far as functionality.  It's just me being paranoid about duplicate records.

Reply:
ok this is new but I don't think it could necessarily be an issue